kihuha/iReporter-db
# Ch refactor test 162530416

#### What does this PR do
contains tests for red_flag models and API

#### Description of the tasks to be completed
- Refactor all unit tests
- Create a separate file for test data

#### How should this be manually tested
How should this be manually tested?
After cloning this repository, navigate to branch ch-refactor-test-162530416. On the terminal window, navigate to the root folder directory and do the following 

1. Install a virtual environment `pip install virtualenv` and activate it (optional but recommended)
2. Install all dependencies by typing `pip install -r requirements.txt`
3. Run the tests by typing either:
- `pytest` -> runs test 
- `pytest --cov-report term-missing --cov=app/` -> runs test with coverage reporting

#### Relevant PT stories
#162530416
